pkgs/build-support/kernel/modules-closure.sh

```diff
@@ -68,7 +68,11 @@ done
 
 mkdir -p $out/lib/firmware
 for module in $(cat closure); do
-    for i in $(modinfo -F firmware $module); do
+    for i in $(modinfo -b $kernel --set-version "$version" -F firmware $module); do
+        # $ modinfo -F firmware unix
+        # name:           unix
+        if test "$i" = "name:"; then break; fi
```

Comment:
Once modinfo patches are merged, we still need this in:
`-b $kernel --set-version "$version" `
```suggestion
```